donkeycar/utilities/TrackSpeedPlanner/deploy-pi-tornado.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Tornado-based Raspberry Pi deployment script for Path Data Visualizer & Editor
+# Uses Python instead of Node.js for better Pi compatibility
+
+set -e
+
+echo "🐍 Setting up Path Data Visualizer on Raspberry Pi (Python/Tornado)..."
+
+# Clean up any previous installation
+if [ -f "uninstall-pi.sh" ]; then
+    echo "🧹 Cleaning up previous installation..."
+    chmod +x uninstall-pi.sh
+    ./uninstall-pi.sh
+    echo ""
+fi
+
+# Check Python installation
+if ! command -v python3 &> /dev/null; then
+    echo "❌ Python 3 not found. Installing..."
+    sudo apt-get update
+    sudo apt-get install -y python3 python3-pip
+fi
+
+echo "✅ Python version: $(python3 --version)"
+
+# Install Tornado if not available
+echo "📦 Installing Python dependencies..."
```

Comment:
Not needed as tornado is installed in `env`.

Reply:
Given that this does check to see if Tornado is already installed and skips if so, this seems like standard best practice. No harm in leaving it in?